lib/soft_deletion/core.rb

```diff
@@ -92,6 +92,13 @@ def soft_delete_dependencies
       self.class.soft_delete_dependents.map { |dependent| SoftDeletion::Dependency.new(self, dependent) }
     end
 
+    def update_counter_cache_associations(increment=false)
```

Comment:
prefer no default ... users should know what they want ... maybe make the argument `1 / -1`

Reply:
I guess 1/-1 won't really work ...

Reply:
Yeah, `1 / -1` won't work so I just created an enumerator. Its easier to understand/read than passing in some value. How do you feel about that?